CHANGELOG.md

```diff
@@ -4,6 +4,35 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.24.0
+
+This release bumps the [`@sentry/replay`](https://github.com/getsentry/sentry-javascript/blob/master/packages/replay/README.md) package from version 0.x to 7.2.0.
+Along with this version bump, we're introducing a few breaking changes.
+Take a look at the [Replay migration guide](https://github.com/getsentry/sentry-javascript/blob/master/packages/replay/MIGRATION.md) for further information.
+Furthermore, we're now also publishing CDN bundles for the Replay integration.
+The Replay version bump is the result of moving the package into the Sentry JavaScript SDK monorepo which aligns the version with our other JS SDK packages.
+
+- build(replay): Generate Replay CDN bundles (#6391)
```

Comment:
I think I would still remove the replay entries, they muddy this up a bit. IMHO they are not really valuable on their own (esp. as a lot of stuff is missing from replay perspective that was merged before)!

Reply:
Fair enough, my reason for including them was also to highlight the (removal of) deprecations/options refactoring. But I can clean it up as we're linking to the migration docs anyway.

Reply:
[5513967](https://github.com/getsentry/sentry-javascript/pull/6436/commits/5513967c0bb3ddf6a736e14532652f2d4300d406)